vsintegration/src/FSharp.Editor/Diagnostics/DocumentDiagnosticAnalyzer.fs

```diff
@@ -104,6 +104,8 @@ type internal FSharpDocumentDiagnosticAnalyzer() =
             return results
         }
 
+    override __.Priority = 10 // Default = 50
```

Comment:
I think we should make this 0, as that is the highest priority available. (In Roslyn, compiler is -2 and 'regular diagnostics' are -1.)